src/components/macroview/rightPanel/editForms/KeyPressForm.tsx

```diff
@@ -35,13 +51,34 @@ export default function KeyPressForm({
     )
       return
 
-    const typeString = selectedElement.data.key_type as keyof typeof KeyType
+    const typeString = selectedElement.data.keytype as keyof typeof KeyType
     setKeypressType(KeyType[typeString])
     setKeypressDuration(selectedElement.data.press_duration.toString())
     setHeadingText(
-      `Key ${HIDLookup.get(selectedElement.data.keypress)?.displayString}`
+      <HStack justifyContent="center" p={1}>
+        <Text>Editing element</Text>
+        <Box
+          h="32px"
+          w="fit-content"
+          bg={bg}
+          border="1px solid"
+          py={1}
+          px={3}
+          borderColor={kebabColour}
+          rounded={borderRadiusStandard}
+        >
+          <Text
+            fontSize={['sm', 'md', 'md']}
+            w="fit-content"
+            whiteSpace="nowrap"
+            fontWeight="bold"
+          >
+            {HIDLookup.get(selectedElement.data.keypress)?.displayString}
+          </Text>
+        </Box>
+      </HStack>
     )
-  }, [selectedElement])
+  }, [bg, kebabColour, selectedElement])
```

Comment:
I think instead of putting this element inside a `useState` and then wrapping it with a `Text` component below, you should just replace the `Text` component with this `HStack`

Reply:
[5895e6d](https://github.com/WootingKb/wooting-macros/pull/188/commits/5895e6db1088b2dc00257b6592ac5f6618f4564f)